CHANGELOG.md

```diff
@@ -2,6 +2,16 @@
 
 ## 0.20
 
+- add `{QCheck,QCheck2}.Gen.{bytes_size,bytes,bytes_of,bytes_printable,small_bytes}`
+- add `QCheck.{Print,Shrink,Observable}.bytes`
+- add `QCheck2.{Print,Shrink}.bytes`
+- add `QCheck.{bytes_gen_of_size,bytes_gen,bytes,small_bytes,bytes_of_size,printable_bytes,printable_bytes_of_size,numeral_bytes,numeral_bytes_of_size}`
```

Comment:
These names are also off.
AFAICS, this adds 6 `arbitrary` `bytes` combinators - all prefixed with `bytes_` - so the last four should be scrapped and `small_bytes` renamed.

Reply:
Some of these are also repeated in line 12:
```
- add `QCheck.{bytes_gen_of_size,bytes_gen,bytes,bytes_small,bytes_of_size,bytes_printable}`
```